### ISSUE TITLE
Cannot download dataset

### ISSUE THREAD

**Issue by enesduran** [leolyliu/TACO-Instructions#5]
Hey, I could not download the dataset. Run into the following 
![Uploading Screenshot from 2024-09-16 21-13-26.png…]()

**Comment by leolyliu** [comment]
Hi enesduran,
We can download the files successfully. Have you solved the problem? Maybe you can wait for some time and download them.

**Comment by leolyliu** [comment]
We have prepared a copy of the dataset at [TACO Dataset V1](https://www.dropbox.com/scl/fo/8w7xir110nbcnq8uo1845/AOaHUxGEcR0sWvfmZRQQk9g?rlkey=xnhajvn71ua5i23w75la1nidx&st=9t8ofde7&dl=0).

**Comment by lan-creator** [comment]
Hello, for the TACO Dataset V1 I still could not install it. It always fails when I download about 
the half size of files and I guess this is the problem of Dropbox when installing large files. Could you give some other method like huggingface, google drive, etc. ? Thank you!

**Comment by leolyliu** [comment]
> Hello, for the TACO Dataset V1 I still could not install it. It always fails when I download about the half size of files and I guess this is the problem of Dropbox when installing large files. Could you give some other method like huggingface, google drive, etc. ? Thank you!

Hi @lan-creator ,
Have you downloaded the dataset now? It's a burden for us to maintain a copy of the dataset on other cloud storages. If you are in Beijing, I can probably send you a copy of the dataset on hard disks.

**Comment by ruoyiqiao** [comment]
Hi @leolyliu,  

I’m experiencing the same issue while downloading the TACO Dataset V1—the download consistently fails around halfway for both the original and backup links, likely due to Dropbox limitations. Have you found a workaround for this?  

Additionally, @lan-creator, did you manage to resolve this issue? If so, I’d really appreciate it if you could share your solution.  

Thanks in advance!

**Comment by leolyliu** [comment]
> Hi [@leolyliu](https://github.com/leolyliu),
> 
> I’m experiencing the same issue while downloading the TACO Dataset V1—the download consistently fails around halfway for both the original and backup links, likely due to Dropbox limitations. Have you found a workaround for this?
> 
> Additionally, [@lan-creator](https://github.com/lan-creator), did you manage to resolve this issue? If so, I’d really appreciate it if you could share your solution.
> 
> Thanks in advance!

Hi @ruoyiqiao,

Sorry for the inconvenience. The backup link has only 20GB of bandwidth per day, and the original link has 1TB of bandwidth per day. Exceeding the bandwidth will disable the data link. Our Dropbox subscriptions make it hard to enlarge the bandwidths.

**Comment by lan-creator** [comment]
> > > Hi [@leolyliu](https://github.com/leolyliu),
> > > I’m experiencing the same issue while downloading the TACO Dataset V1—the download consistently fails around halfway for both the original and backup links, likely due to Dropbox limitations. Have you found a workaround for this?
> > > Additionally, [@lan-creator](https://github.com/lan-creator), did you manage to resolve this issue? If so, I’d really appreciate it if you could share your solution.
> > > Thanks in advance!
> > 
> > 
> > Hi [@ruoyiqiao](https://github.com/ruoyiqiao),
> > Sorry for the inconvenience. The backup link has only 20GB of bandwidth per day, and the original link has 1TB of bandwidth per day. Exceeding the bandwidth will disable the data link. Our Dropbox subscriptions make it hard to enlarge the bandwidths.
> 
> Hi [@leolyliu](https://github.com/leolyliu),
> 
> Thanks for response. Does the download failing halfway mean that the main link's bandwidth has already been exceeded? In my case, I can still start the download, but it always stops automatically once the progress reaches ~10GB.
> 
> Thanks again!

Yeah, I encountered the same problem and I guess this is caused by dropbox. I also encounter this problem when I use dropbox to download other datasets. So I'll really appreciate it if there is any other cloud storage.

**Comment by leolyliu** [comment]
> > > Hi [@leolyliu](https://github.com/leolyliu),
> > > I’m experiencing the same issue while downloading the TACO Dataset V1—the download consistently fails around halfway for both the original and backup links, likely due to Dropbox limitations. Have you found a workaround for this?
> > > Additionally, [@lan-creator](https://github.com/lan-creator), did you manage to resolve this issue? If so, I’d really appreciate it if you could share your solution.
> > > Thanks in advance!
> > 
> > 
> > Hi [@ruoyiqiao](https://github.com/ruoyiqiao),
> > Sorry for the inconvenience. The backup link has only 20GB of bandwidth per day, and the original link has 1TB of bandwidth per day. Exceeding the bandwidth will disable the data link. Our Dropbox subscriptions make it hard to enlarge the bandwidths.
> 
> Hi [@leolyliu](https://github.com/leolyliu),
> 
> Thanks for response. Does the download failing halfway mean that the main link's bandwidth has already been exceeded? In my case, I can still start the download, but it always stops automatically once the progress reaches ~10GB.
> 
> Thanks again!

Hi @ruoyiqiao @lan-creator ,

This is not the case bandwidth exceeds but probably due to network issues. I have downloaded the data several times and haven't encountered this issue. Maybe changing the network setup or the VPN could solve it. We are finding other cloud disks and will upload the dataset there.
